e2e/flags/.snapshots/TestReportFlagsShouldFail-invalid-context-flag

```diff
@@ -30,6 +30,7 @@ Scan Flags
       --external-rule-dir strings            Specify directories paths that contain .yaml files with external rules configuration
       --force                                Disable the cache and runs the detections again
       --internal-domains strings             Define regular expressions for better classification of private or unreachable domains e.g. --internal-domains=".*.my-company.com,private.sh"
+      --parallel int                         Specify the amount of parallelism to use during the scan (default 2)
```

Comment:
```suggestion
      --parallel int                         Specify the amount of parallelism to use during the scan (default 4)
```

Reply:
Why 2 here (while the default seems to be 4)?

Reply:
The default is `min(cpus, 4)`. Locally I get 4, under CI it's 2.

It's a similar issue with those status codes, different output locally. The E2E tests are aren't great in that way.